tests/queries/0_stateless/03164_parts_filter_profiles_test.sql

```diff
@@ -0,0 +1,25 @@
+set log_queries=1;
+set log_queries_min_type='QUERY_FINISH';
+
```

Comment:
The test name is too generic, developers will have no clue about the purpose of the test by looking only at its file name and will have to read it to understand. It also lacks a basic comment what is being tested. Ideally, please rename to something less generic. Please add a comment here, e.g. 

```sql
-- Tests ProfileEvents "SelectedPartsByPartitionKey", "SelectedPartsByPartitionKey", "SelectedMarksByPrimaryKey"